marlowe-dashboard-client/src/Template/View.purs

```diff
@@ -100,7 +100,14 @@ contractNicknameDisplay contractName contractNickname =
         [ div
             [ classNames [ "max-w-sm", "mx-auto", "px-4", "pt-2" ] ]
             [ input
-                [ classNames $ (Css.input $ null contractNickname) <> [ "bg-transparent", "font-semibold" ]
+                [ classNames
+                    -- TODO: Once we remove the readOnly, remove this filter. I tried adding "text-black" to the end of the array
+                    
+                    --       but the browser does not respect ordering and for some reason "text-darkgray was winning"
+                    
```

Comment:
remove the extra whitespace

Reply:
tha is added by purty for some reason 🤷‍♂️